.travis.yml

```diff
@@ -17,7 +25,7 @@ deploy:
     local_dir: build
     detect_encoding: true
     on:
-        branch: develop
+        branch: Bjorn248/feature/issue-54
```

Comment:
@Bjorn248 Is this right? Seems like it should be `develop` and not a feature branch.